lib/delta_tables_generator.rb

```diff
@@ -0,0 +1,59 @@
+require 'logger'
+
+require 'active_support/notifications'
+require 'active_support/log_subscriber'
+
+module DeltaTablesGenerator
+  extend self
+
+  delegate :instrument, :subscribe, to: ActiveSupport::Notifications
+
+  # Starts the generation of the deltas data.
+  def generate(day: Date.current)
+    DeltaTablesBackend.with_redis_lock do
```

Comment:
How comes we need to lock the redis table when generating the change events?

Reply:
It's a leftover from an initial version, I'll remove it, thanks for pointing it out :)